### DISCUSSION TITLE
bin.entware.net - CF Error 522 Parts of Australia

### DISCUSSION THREAD

**Discussion by XxUnkn0wnxX** [Entware/Entware#822] [Q&A] [answered]
Hi, I'm having trouble accessing bin.entware.net within AU, is this a permanent issue? as I'm trying to setup packages on my router & I cannot download any packages due to this.

I've tested via a VPN & it's some states that this issue is happening on.
<img width="1252" alt="1" src="https://user-images.githubusercontent.com/2242880/179503775-39db73ab-d2d2-4b92-9361-cacf821b3f74.png">
<img width="1076" alt="2" src="https://user-images.githubusercontent.com/2242880/179503817-605df9e9-97a7-4ee9-a73b-e28e64271710.png">
<img width="1252" alt="3" src="https://user-images.githubusercontent.com/2242880/179503888-e32dbdb6-937d-4259-9f7c-941195501cad.jpg">

**Edit:**
issue resolved, flushed DNS caches..

**Comment by ryzhovau** [comment, marked as answer]
It's all about CloudFlare. Most of complains are from Australia and China.
